SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The indicated versions below are currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We acknowledge that every line of code that we write may potentially contain security issues.
+We are committed to address issues responsibly and provide patches as quickly as possible.
+Please contact us via [E-Mail (opensource+repo@civicactions.com)](mailto:opensource+repo@civicactions.com?subject=CivicActions&#x3a;%20Security&body=Security%20description&#x3a;) immediately after discovery of the vulnerability. Thanks!
```

Comment:
@mgifford I believe the current plan is to use the existing `security` list rather than the new `opensource` list.